tests/Gedmo/Sluggable/Fixture/Issue100/Article.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Doctrine Behavioral Extensions package.
+ * (c) Gediminas Morkevicius <gediminas.morkevicius@gmail.com> http://www.gediminasm.org
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Gedmo\Tests\Sluggable\Fixture\Issue100;
+
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+use Gedmo\Sluggable\Sluggable;
+use Gedmo\Translatable\Translatable;
+
+/**
+ * @ORM\Entity
+ */
+#[ORM\Entity]
+class Article implements Sluggable, Translatable
+{
+    /**
+     * @var int|null
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: Types::INTEGER)]
+    private $id;
+
+    /**
+     * @var string|null
+     *
+     * @Gedmo\Translatable
+     * @ORM\Column(name="title", type="string", length=64)
+     */
+    #[ORM\Column(name: 'title', type: Types::STRING, length: 64)]
+    #[Gedmo\Translatable]
+    private $title;
+
+    /**
+     * @var string|null
+     *
+     * @Gedmo\Translatable
+     * @Gedmo\Slug(separator="-", updatable=true, fields={"title"}, unique=true)
```

Comment:
```suggestion
     * @Gedmo\Slug(separator="-", updatable=true, fields={"title"}, unique=true, uniqueOverTranslations=true)
```

Otherwise the PHP 7.4 build using annotations will have a different behavior from the PHP 8 builds with attributes